src/app/trainees/state/trainees.state.spec.ts

```diff
@@ -107,13 +107,13 @@ describe("Trainees state", () => {
 
   it("should dispatch 'UnderNoticeFilter' and update store", () => {
     store.dispatch(new UnderNoticeFilter());
-    const underNotice = store.selectSnapshot(TraineesState.underNotice);
-    expect(underNotice).toBeTruthy();
+    const filter = store.selectSnapshot(TraineesState.filter);
+    expect(filter).toEqual("underNotice");
```

Comment:
Can we serve all this strings("underNotice", "allDoctors") from a const, so that it will be cleaner and reusable.
Apologies for not raising this in earlier comments.

Reply:
not a problem @babul3012 I appreciate the feedback, I have converted the string type to an enum instead. pr updated